core/consensus/consensus_event.hpp

```diff
@@ -11,11 +11,12 @@ namespace consensus_event {
 
 struct ConsensusEvent {
     std::unique_ptr<abstract_transaction::AbstractTransaction> tx;
-    std::vector<std::string> txSignatures;
+    std::vector<std::string> signatures;
     std::string merkleRoot;
+    unsigned long long order;
     std::vector<std::string> merkleRootSignatures;
 
-    ConsensusEvent(std::unique_ptr<abstract_transaction::AbstractTransaction> tx){
+    ConsensusEvent(std::unique_ptr<abstract_transaction::AbstractTransaction> tx) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Member variable 'ConsensusEvent::order' is not initialized in the constructor.](https://www.codacy.com/app/soramitsu/iroha/file/4256044911/issues/source?bid=3712501&fileBranchId=3831026#l19)